test/dataframerow.jl

```diff
@@ -304,6 +304,18 @@ end
 end
 
 @testset "show" begin
+    function capture_stdout(f::Function)
+        oldstdout = stdout
+        rd, wr = redirect_stdout()
+        f()
+        str = String(readavailable(rd))
```

Comment:
I would write it as:

```
        redirect_stdout(oldstdout)
        size = displaysize(rd)
        close(wr)
        str = read(rd, String)
        close(rd)
```
as it will be non-blocking when `f` produces no output to `stdout`. Probably a minor thing.

Reply:
TBH I've just copied the code from show.jl.

Reply:
This is a minor thing, but I just wanted to make sure you know the problem with the current implementation (you can check that it is blocking if you e.g. run `capture_stdout(()->nothing)`).

So maybe let us fix it in both places (as users might copy the code from DataFrames.jl for different purposes thinking it is a well written package 😄).

Reply:
OK, I'm quite bad at streaming so I've just adopted your code. :-)

Reply:
Actually streaming in Julia is rather tricky from my experience but what I have proposed seems to work better.